src/Integration.Vsix/Analysis/AnalysisConfigMonitor.cs

```diff
@@ -72,24 +75,10 @@ private void OnUserSettingsChanged(object sender, EventArgs e)
             }
         }
 
-        private void OnSolutionBindingUpdated(object sender, EventArgs e)
+        private void OnSuppressionsUpdated(object sender, EventArgs e)
         {
-            // NB assumes exception handling is done by the AnalysisRequester
```

Comment:
Error handling in event handlers in VS is important as events are often on the UI thread so an unhandled exception will crash VS. This comment captures the design decision that these event handlers deliberately are not handling errors.

If you don't like the comment, do something else to make this clear e.g. rename the method to `UnsafeHandleSuppresionsUpdated` (and in that case rename `OnUserSettingsChanged` for consistency).